onboarding/FIRST_PULL_REQUEST.md

```diff
@@ -11,45 +11,41 @@
 
 ### 1. Clone the repository using Git and create a new branch
 
-#### For the first part, it is necessary to clone the repository where you will work on your local PC. For this, open Git and type the following command:
+For the first part, it is necessary to clone the repository where we will work on your local PC. For this, open Git and type the following command:
 
-#### *git clone "repository of your choice"* 
+*git clone "repository of your choice"* 
```

Comment:
Add code block for all commands examples. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks